src/alert.cpp

```diff
@@ -245,8 +247,22 @@ bool CAlert::ProcessAlert(const std::vector<unsigned char>& alertKey, bool fThre
     return true;
 }
 
-void
-CAlert::Notify(const std::string& strMessage, bool fThread)
+// static
+void CAlert::NotifyInternal(const std::string& strMessage)
+{
+    static CAlert internalAlert;
+    internalAlert.vchMsg = std::vector<unsigned char>(strMessage.begin(), strMessage.end());
```

Comment:
`internalAlert.vchMsg` will never be shown in your case. What's the reason in instantiating a CAlert? IMO it's completely unused.